### PR TITLE
github: make use of LXD's actions/lp-snap-build in build job

### PR DESCRIPTION
**Pull request by simondeziel**

### CODE REVIEW

File: .github/workflows/builds.yml [hunk 1]
```diff
@@ -16,51 +16,35 @@ jobs:
     name: Trigger snap build
     runs-on: ubuntu-24.04
     if: ${{ github.repository == 'canonical/lxd-pkg-snap' && github.actor != 'dependabot[bot]' }}
+    env:
+      SSH_AUTH_SOCK: /tmp/ssh_agent.sock
+      PACKAGE: "lxd"
+      REPO: "git+ssh://lxdbot@git.launchpad.net/~canonical-lxd/lxd"
+      BRANCH: >-
+        ${{ fromJson('{
+          "main": "latest-edge",
```
Comment (tomponline) [inline .github/workflows/builds.yml]: there's no main branch in this repo, you can just use the branch name

Comment (simondeziel) [reply .github/workflows/builds.yml]: Thanks, fixed

**Review by tomponline — DISMISSED** [review]

**Comment by tomponline** [comment]
needs a rebase please

**Comment by simondeziel** [comment]
> needs a rebase please

Done

**Review by tomponline — APPROVED** [review]

**Comment by tomponline** [comment]
@simondeziel please may you create PRs for the other branches, as there are some subtle differences for the candidate branches that i dont want to trust to a backport i do some point in the future.

**Comment by simondeziel** [comment]
@tomponline I went all the way back to `5.0-candidate`. Should I also do `4.0-{edge,candidate}`?

**Comment by tomponline** [comment]
> @tomponline I went all the way back to `5.0-candidate`. Should I also do `4.0-{edge,candidate}`?

yes please as we still need to build those snaps from time to time

**Comment by tomponline** [comment]
@simondeziel can lxd-imagebuilder also use it?

https://github.com/canonical/lxd-imagebuilder/blob/main/.github/workflows/tests.yml#L58-L110

**Pull request merged**